app/lib/bk/compat/parsers/bitbucket/services.rb

```diff
@@ -1,8 +1,5 @@
 # frozen_string_literal: true
 
-require_relative '../../pipeline/step'
-require_relative '../../pipeline/plugin'
-
```

Comment:
This file still uses some classes so it should include the dependencies explicitly (assuming you also take the other suggestion on this file to change the CircleCI-specific command step to the generic one):

```suggestion
require_relative '../../models/plugin'
require_relative '../../models/steps/command'

```

Reply:
Added back in 👍